.teamcity/src/Common.kt

```diff
@@ -0,0 +1,24 @@
+const val PROJECT_BRANCH = "master_teamcity"
```

Comment:
Should we change this before merging?

Reply:
Ugh, I'll dig and see if I can figure out a better way to handle this. The only way that I know of (using DSL context parameters) has to be done through the UI.